assets/js/base/components/reviews/review-list/index.tsx

```diff
@@ -11,17 +11,30 @@ import ReviewListItem from '../review-list-item';
 import type { Review } from '../types';
 import './style.scss';
 
+export type ReviewListAttributes = BlockAttributes & {
+	imageType: string;
```

Comment:
One thing I'm not sure about is how we should type the block attributes,

With blocks I think attributes should all be `?` technically, but if we pass a lot of fallback defaults for attributes maybe these will always be present

Reply:
I think we should type those attributes according to how we register them in `assets/js/blocks/reviews/attributes.js`.

By the way, I also think we should export the attribute type from the block directory then import it here, it's easier to update the type in the future.